cmd/fortifyExecuteScan.go

```diff
@@ -123,6 +126,14 @@ func determineArtifact(config fortifyExecuteScanOptions, utils fortifyUtils) (ve
 func runFortifyScan(config fortifyExecuteScanOptions, sys fortify.System, utils fortifyUtils, telemetryData *telemetry.CustomData, influx *fortifyExecuteScanInflux, auditStatus map[string]string) ([]piperutils.Path, error) {
 	var reports []piperutils.Path
 	log.Entry().Debugf("Running Fortify scan against SSC at %v", config.ServerURL)
+	_, err := execInPath("fortifyupdate")
+	if err != nil {
+		return reports, fmt.Errorf("ERROR , command not found: fortifyupdate. Please configure a supported docker image or install Fortify SCA on the system.")
+	}
+	_, err = execInPath("sourceanalyzer")
+	if err != nil {
+		return reports, fmt.Errorf("ERROR , command not found: fortifyupdate. Please configure a supported docker image or install Fortify SCA on the system.")
```

Comment:
Here we are testing the sourceanalyzer but reporting a missing fortifyupdate.

Reply:
my bad